### PR TITLE
Update devtools-extension with a note about it not working

### PR DESCRIPTION
**Pull request by kevinsimper**

### CODE REVIEW

File: docs/tutorial/devtools-extension.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ the `BrowserWindow.addDevToolsExtension` API to load them. The loaded extensions
 will be remembered so you don't need to call the API every time when creating
 a window.
 
+## NOTE: React DevTools does not work, follow the issue here https://github.com/atom/electron/issues/915
```
Comment (zcbenz) [inline docs/tutorial/devtools-extension.md]: I'm 👍 on this, but showing it as a title is probably too strong, maybe we can just show it in bold:

```markdown
** Note: ... **
```

Comment (kevinsimper) [inline docs/tutorial/devtools-extension.md]: But you should not be able to miss it, it is not working so maybe the react example should be removed instead?

Comment (zcbenz) [inline docs/tutorial/devtools-extension.md]: How about using other devtools extension as example like https://github.com/mauricecruz/chrome-devtools-zerodarkmatrix-theme?

**Comment by kevinsimper** [comment]
I have updated it to be a note, good idea with making it to another tutorial, I just have not tried that.

**Comment by zcbenz** [comment]
👍

**Pull request merged**